### PR TITLE
755 update the equality and diversity survey

### PR DESCRIPTION
**Pull request by tomas-stefano**
## Context

Update the Equality and diversity survey so that back links and "Continue" buttons take you back to the review screen when changing an answer.

Update the check your answers page in our HESA questionnaire (to not use brackets)

### CODE REVIEW

File: app/controllers/candidate_interface/equality_and_diversity_controller.rb [hunk 1]
```diff
@@ -44,7 +44,7 @@ def update_ethnic_group
         if ethnic_group_param == 'Prefer not to say'
           redirect_to free_school_meals_or_review(current_application)
         else
-          redirect_to candidate_interface_edit_equality_and_diversity_ethnic_background_path
+          redirect_to candidate_interface_edit_equality_and_diversity_ethnic_background_path(params[:return_to].present? ? params.slice(:return_to) : nil)
```
Comment (carlosmartinez) [inline app/controllers/candidate_interface/equality_and_diversity_controller.rb:47]: does `params.fetch(:return_to, nil)` do the job?

**Review by frankieroberto — CHANGES_REQUESTED** [review]
I'm getting a 500 error when clicking 'Change ethnicity' and then pressing the Continue button.

When clicking 'Change' on any of the answers, the 'Back' link should take you back to the Check your answers page.

**Review by frankieroberto — CHANGES_REQUESTED** [review]
Tested in review app. Back links are working now when used after changing an answer.

2 minor amends please:

* change "Disability" to "Disabilities or health conditions" on Check your answers page
* If someone selects "I do not have any of these disabilities or health conditions", display that exact text on the Check your answers page instead of "No".

**Review by carlosmartinez — APPROVED** [review]

**Review by JonF1 — APPROVED** [review]

File: app/controllers/candidate_interface/equality_and_diversity_controller.rb [hunk 2]
```diff
@@ -137,5 +145,9 @@ def redirect_to_review_unless_ready_to_submit
     def ready_to_submit?
       @ready_to_submit ||= CandidateInterface::ApplicationFormPresenter.new(current_application).ready_to_submit?
     end
+
+    def review_back_link
```
[the remark this answers is not in the document]
Comment (tomas-stefano) [reply app/controllers/candidate_interface/equality_and_diversity_controller.rb]: Done ♻️

**Review by frankieroberto — APPROVED** [review]

**Pull request merged**